lib/seed_helper.rb

```diff
@@ -158,12 +158,6 @@ def format_errors(model = nil)
       model.errors.full_messages.join(';')
     end
 
-    private
-
-    def logger
-      Foreman::Logging.logger('app')
-    end
-
     def test_template_requirements(template_name, requirements)
       requirements.each do |r|
```

Comment:
While looking at this method my mind was rewriting this to `.all?` to avoid the explicit `return` statements.